### PR TITLE
fix long lines, turn on rubocop in travis

### PR DESCRIPTION
**Pull request by sandlerr**
### Risks

- [medium] failing app uploads

### CODE REVIEW

**Comment by sandlerr** [comment]
cc @zendesk/vegemite @zendesk/wombat

File: lib/zendesk_apps_support/package.rb [hunk 1]
```diff
@@ -226,13 +223,13 @@ def translations
...
           locale = File.basename(path, File.extname(path))
 
           locale_translations = if locale == manifest.default_locale
-            default_translations
-          else
-            deep_merge_hash(default_translations, process_translations(path))
+                                  default_translations
+                                else
+                                  deep_merge_hash(default_translations, process_translations(path))
           end
```
Comment (danielbreves) [inline lib/zendesk_apps_support/package.rb]: Can you indent the `end` please?

File: lib/zendesk_apps_support/package.rb [hunk 2]
```diff
@@ -272,17 +269,25 @@ def location_icons
...
+                                                 specific_png = "icon_#{location}_#{state}.png"
+                                                 icon_state_hash[state] = if has_file?("assets/#{specific_png}")
+                                                                            specific_png
+                                                                          else
+                                                                            inactive_png
+                                                                          end
+                                               end
+                                             end
+                                           else
+                                             {}
```
Comment (danielbreves) [inline lib/zendesk_apps_support/package.rb]: Not sure this is better tbh. Could we extract it to a method?

File: lib/zendesk_apps_support/finders.rb [hunk 3]
```diff
@@ -12,7 +13,7 @@ def find_by(arg)
 
     def find_by!(arg)
       found = find_by(arg)
-      fail RecordNotFound.new("Unable to find #{self.name} with #{arg.inspect}") if found.nil?
+      raise RecordNotFound, "Unable to find #{name} with #{arg.inspect}" if found.nil?
```
Comment (danielbreves) [inline lib/zendesk_apps_support/finders.rb]: Could you add parenthesis between the args?

**Comment by sandlerr** [comment]
@danielbreves 💇‍♂️

**Pull request merged**